src/sage/schemes/elliptic_curves/ell_curve_isogeny.py

```diff
@@ -889,15 +889,15 @@ class EllipticCurveIsogeny(EllipticCurveHom):
     Examples over relative number fields used not to work (see :trac:`16779`)::
 
         sage: pol26 = hilbert_class_polynomial(-4*26)
-        sage: pol = NumberField(pol26,'a').optimized_representation()[0].polynomial()   # optional - sage.rings.number_field
-        sage: K.<a> = NumberField(pol)                                                  # optional - sage.rings.number_field
-        sage: j = pol26.roots(K)[0][0]                                                  # optional - sage.rings.number_field
-        sage: E = EllipticCurve(j=j)                                                    # optional - sage.rings.number_field
-        sage: L.<b> = K.extension(x^2 + 26)                                             # optional - sage.rings.number_field
-        sage: EL = E.change_ring(L)                                                     # optional - sage.rings.number_field
-        sage: iso2 = EL.isogenies_prime_degree(2); len(iso2)                            # optional - sage.rings.number_field
+        sage: pol = NumberField(pol26,'a').optimized_representation()[0].polynomial()   # long time  # optional - sage.rings.number_field
```

Comment:
It would fit if one first goes `sage: F = NumberField(pol26,'a')`

Reply:
Ok, done. But this is neverending. Please let's try to finish here.